### ISSUE TITLE
Possible bug in spaceAroundOperators

### ISSUE THREAD

**Issue by meherkasam** [nicklockwood/SwiftFormat#732]
My test example:
```
func myFunc() -> [UIColor] {
    return [ .red ]
}
```

The command I ran:
`$ swiftformat test.swift --rules spaceAroundOperators`

I expected the code to remain unchanged. However, it updates to the following:
```
func myFunc() -> [UIColor] {
    return [.red ]
}
```

It appears to be treating the `[` as an operand.

**Comment by nicklockwood** [comment]
@meherkasam thanks for reporting, I think the bug is actually that it always tries to remove the space between an opening bracket and a prefix `.`, but either way it needs fixing.

**Comment by meherkasam** [comment]
Thank you for the quick response!

**Comment by nicklockwood** [comment]
@meherkasam fixed in 0.46.2.
